manifests/init.pp

```diff
@@ -451,6 +455,7 @@
   Optional[Stdlib::Absolutepath] $source_config_file              = undef,
   Boolean                        $store_git_keys_in_db            = false,
   Optional[Hash]                 $unicorn                         = undef,
+  Optional[Hash]                 $puma                            = undef,
```

Comment:
is there any chance to enforce the datatype for the hash key and value?

Reply:
Not really, because we would have to keep up with the changes done in the Omnibus package/Chef cookbook.